app/controllers/rb_application_controller.rb

```diff
@@ -39,6 +39,8 @@ class RbApplicationController < ApplicationController
 
   before_filter :load_sprint_and_project, :check_if_plugin_is_configured, :authorize
 
+  skip_before_action :verify_authenticity_token, if: -> { Rails.env.test? }
```

Comment:
@myabc what is this required for? I assume that you introduced this to not run into something like http://api.rubyonrails.org/v4.1.6/classes/ActionController/RequestForgeryProtection.html. But this can be circumvented differenty, e.g. by providing the csrf token 1bc34e5fc4ebc9dbd2fc4fa96188633aa4b89b59 (it does exist in specs, right?).